src/hooks/screen/useChatScreenHook.ts

```diff
@@ -33,7 +33,7 @@ interface ScrollContextProps {
 }
 export const ScrollContext = React.createContext<ScrollContextProps | null>(null);
 
-function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
+function useChatScreenHook(type?: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `type` parameter in the `useChatScreenHook` function has been made optional. This could potentially alter its usage throughout the codebase. If there are places in the code where this hook is called without a `type` argument, it might lead to unexpected behavior if the function implementation assumes that `type` will always be defined. Please ensure that the function handles the case when `type` is undefined.

```diff
- function useChatScreenHook(type?: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
+ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS' = 'SIGNED'): UseChatScreenHook {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->